### PR TITLE
Add art

### LINKED ISSUE #72: [ENH] ascii logo at startup in console
sweety :D

### PR DESCRIPTION
**Pull request by SupinePandora43**
image
closes #72

### CODE REVIEW

File: gm_dotnet_managed/GmodNET/Startup.cs [hunk 1]
```diff
@@ -166,6 +166,18 @@ internal static class Startup
...
+                lua.GetField(-1, "print");
+                lua.PushString(
+                    "   _____       _   _   ______   _______ \n"+
+                    "  / ____|     | \\ | | |  ____| |__   __|\n"+
+                    " | |  __      |  \\| | | |__       | |   \n"+
+                    " | | |_ |     | . ` | |  __|      | |   \n"+
+                    " | |__| |  _  | |\\  | | |____     | |   \n"+
+                    "  \\_____| (_) |_| \\_| |______|    |_|   \n");
+                lua.MCall(1,0);
+                lua.Pop();
```
Comment (AaronJThompson) [inline gm_dotnet_managed/GmodNET/Startup.cs]: .Pop(1) please if possible to follow the rest of the calling conventions

File: gm_dotnet_managed/GmodNET/Startup.cs [hunk 2]
```diff
@@ -166,6 +166,18 @@ internal static class Startup
...
+                lua.PushSpecial(SPECIAL_TABLES.SPECIAL_GLOB);
+                lua.GetField(-1, "print");
+                lua.PushString(
+                    "   _____       _   _   ______   _______ \n"+
+                    "  / ____|     | \\ | | |  ____| |__   __|\n"+
+                    " | |  __      |  \\| | | |__       | |   \n"+
+                    " | | |_ |     | . ` | |  __|      | |   \n"+
+                    " | |__| |  _  | |\\  | | |____     | |   \n"+
+                    "  \\_____| (_) |_| \\_| |______|    |_|   \n");
+                lua.MCall(1,0);
```
Comment (AaronJThompson) [inline gm_dotnet_managed/GmodNET/Startup.cs]: Can you add a space after the `, `

**Review by AaronJThompson — CHANGES_REQUESTED** [review]

File: gm_dotnet_managed/GmodNET/Startup.cs (hunk 1 above)
Comment (SupinePandora43) [reply gm_dotnet_managed/GmodNET/Startup.cs]: resolved in https://github.com/GmodNET/GmodDotNet/pull/74/commits/e3471759fc7e8196e0b0fd4be3fd1add3a6b807b

File: gm_dotnet_managed/GmodNET/Startup.cs (hunk 2 above)
Comment (SupinePandora43) [reply gm_dotnet_managed/GmodNET/Startup.cs]: resolved in https://github.com/GmodNET/GmodDotNet/pull/74/commits/e3471759fc7e8196e0b0fd4be3fd1add3a6b807b

**Review by AaronJThompson — APPROVED** [review]
Approved. Thanks @SupinePandora43!

**Pull request merged**